apps/discord-bot/src/listeners/parity/cache-parity.test.ts

```diff
@@ -31,9 +29,7 @@ beforeEach(async () => {
 	client = await setupAnswerOverflowBot();
 	guild = mockGuild(client);
 	member = mockGuildMember({ client, guild });
-	user = await prisma.user.create({
-		data: {},
-	});
+	user = await db.insert(users).values({}); // TODO: set values (prisma -> drizzle)
```

Comment:
we should maybe make this not a leaky abstraction and just make a func in the  db package